test/issue1099-empty-settings-json.sh

```diff
@@ -0,0 +1,25 @@
+#!/usr/bin/env bash
+
+if [ -e /var/lib/dub/settings.json ]; then
```

Comment:
Please set `HOME` explicitly in here in order to avoid messing with system wide configs

Reply:
Sorry, but setting `HOME` to a temporary directory will drastically simplify this and you can run the test anywhere ;-)

```
export HOME=some/temp/dir 
```

Reply:
Fixed this up :)